### PR TITLE
Update the namespace-specific-policy example to demonstrate the automated rendering feature

### PR DESCRIPTION
**Pull request by nan-yu**

### CODE REVIEW

**Comment by nan-yu** [comment]
@janetkuo @haiyanmeng @Liujingfang1 
Please review the example when you get a chance.

File: namespace-specific-policy/README.md [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
+One cluster can be shared between multiple tenants. For multi-tenancy, the namespaces used by different tenants should have different policies according to the best practices for multi-tenancy.
+This doc walks you through the steps for configuring namespace specific policies, such as Role, RoleBinding and NetworkPolicy for a cluster shared between tenants.
+
+This example provides two different ways of rendering and syncing your configurations
+- **manual rendering**:
+  this option requires you to render the configurations using the Kustomize CLI manually,
+  and check in the rendered configurations to your Git repository.
+  Config Sync will sync from the rendered output directly.
+- **automated rendering**:
+  Config Sync support automated rendering in versions 1.9.0 or later.
```
Comment (Liujingfang1) [inline namespace-specific-policy/README.md:12]: supports

File: namespace-specific-policy/automated-rendering/README.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+The `automated-rendering` directory includes a root `kustomization.yaml` file that references the three tenant overlays.
+
+```yaml
+resources:
+- configsync-src/tenant-a
+- configsync-src/tenant-b
+- configsync-src/tenant-c
+```
+
+The `automated-rendering/configsync-src` file is a symbolic link that links to the source directory: `../configsync-src`.
```
Comment (Liujingfang1) [inline namespace-specific-policy/automated-rendering/README.md:31]: Have you verified this example work with the symbolic link used here?

Comment (nan-yu) [reply namespace-specific-policy/automated-rendering/README.md:31]: Yes, the `automated-rendering` directory can be synced successfully to a cluster with the symbolic link.

File: namespace-specific-policy/README.md (hunk 1 above)
Comment (nan-yu) [reply namespace-specific-policy/README.md]: DONE

File: namespace-specific-policy/automated-rendering/README.md (hunk 2 above)
Comment (Liujingfang1) [reply namespace-specific-policy/automated-rendering/README.md:31]: Thanks for confirming.

File: namespace-specific-policy/README.md [hunk 3]
```diff
@@ -1,222 +1,49 @@
 # Configuring namespace specific policies
 
-One cluster can be shared between multiple tenants. For multi-tenancy, the namespaces used by different tenants should have different policies according to the best practices for multi-tenancy. This doc walks you through the steps for configuring namespace specific policies, such as Role, RoleBinding and NetworkPolicy for a cluster shared between tenants.
+One cluster can be shared between multiple teams. For multi-tenancy, the namespaces used by different teams should have different policies according to the best practices for multi-tenancy.
+This doc walks you through the steps for configuring namespace specific policies, such as Role, RoleBinding and NetworkPolicy for a cluster shared between teams.
+
+This example provides two different ways of rendering and syncing your configurations
+- **manual rendering**:
```
[the remark this answers is not in the document]
Comment (nan-yu) [reply namespace-specific-policy/README.md]: Link added.

The example details are shared between the manual rendering option and automated rendering option, so I added them here.

File: namespace-specific-policy/README.md [hunk 4]
```diff
@@ -1,222 +1,49 @@
...
+  this option requires you to render the configurations using the Kustomize CLI manually,
+  and check in the rendered configurations to your Git repository.
+  Config Sync will sync from the rendered output directly.
+- **automated rendering**:
```
[the remark this answers is not in the document]
Comment (nan-yu) [reply namespace-specific-policy/README.md]: Link added.

The example details are shared between the manual rendering option and automated rendering option, so I added them here.

File: namespace-specific-policy/README.md [hunk 5]
```diff
@@ -1,222 +1,49 @@
...
+  Config Sync supports automated rendering in versions 1.9.0 or later.
```
[the remark this answers is not in the document]
Comment (nan-yu) [reply namespace-specific-policy/README.md]: DONE

File: namespace-specific-policy/README.md [hunk 6]
```diff
@@ -1,222 +1,49 @@
...
+
+
+The directory `configsync-src` contains the configuration in kustomize format.
+There are one base and three overlays `team-a`, `team-b` and `team-c`.
+Each overlay is a customization of the shared `base`.
+The difference between different overlays is from two parts:
+- Namespace. The configuration inside the directory `configsync-src/<TEAM>` is all in the namespace `<TEAM>`.
+  This is achieved by adding the namespace directive in `configsync-src/<TEAM>/kustomization.yaml`.
+  For example, in `configsync-src/team-a/kustomization.yaml`:
+  Namespace: team-a
```
[the remark this answers is not in the document]
Comment (nan-yu) [reply namespace-specific-policy/README.md]: DONE

File: namespace-specific-policy/README.md [hunk 7]
```diff
@@ -1,222 +1,49 @@
...
-│   │   ├── ~g_v1_namespace_default.yaml
-│   │   ├── networking.k8s.io_v1_networkpolicy_deny-all.yaml
-│   │   ├── rbac.authorization.k8s.io_v1_rolebinding_tenant-admin-rolebinding.yaml
-│   │   └── rbac.authorization.k8s.io_v1_role_tenant-admin.yaml
-│   └── tenant-c
-│       ├── ~g_v1_namespace_default.yaml
-│       ├── networking.k8s.io_v1_networkpolicy_deny-all.yaml
-│       ├── rbac.authorization.k8s.io_v1_rolebinding_tenant-admin-rolebinding.yaml
-│       └── rbac.authorization.k8s.io_v1_role_tenant-admin.yaml
+├── automated-rendering
```
[the remark this answers is not in the document]
Comment (nan-yu) [reply namespace-specific-policy/README.md:40]: DONE

**Pull request merged**